src/Card/CardFlipper.component.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react"
+import styled, { css } from "styled-components"
+import { CardProps } from "./Card.component"
+
+export interface CardFlipperProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Determines if the card flipper will flip vertically.
+   *
+   * @default false
+   * */
+  vertical?: boolean
+  children: React.ReactElement<CardProps>[]
```

Comment:
Any particular reason you needed to add this prop explicitly? `React.FC` should automatically support this prop, right?

Reply:
No longer submitting CardFlipper with the native card component.